<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="signal_group">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>55</y>
     <width>960</width>
     <height>405</height>
    </rect>
   </property>
   <property name="title">
    <string>Signals</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="data_list">
   <property name="geometry">
    <rect>
     <x>975</x>
     <y>40</y>
     <width>171</width>
     <height>411</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="loaddata">
   <property name="geometry">
    <rect>
     <x>975</x>
     <y>10</y>
     <width>170</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>DATA LOAD</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="label_adjust_group">
   <property name="geometry">
    <rect>
     <x>975</x>
     <y>460</y>
     <width>170</width>
     <height>180</height>
    </rect>
   </property>
   <property name="title">
    <string>Label adjust</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X1:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="x1_adjust">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>30</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="end_adjusting">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="x2_adjust">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>70</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X2:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_adjusting">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="signal_view_group_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>560</y>
     <width>960</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Signal view</string>
   </property>
   <widget class="QSpinBox" name="start_point">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="end_point">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="step">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="local_view">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Local</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start point:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>End point:</string>
    </property>
   </widget>
   <widget class="QLabel" name="signal_view_group">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="first_setting_group">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>460</y>
     <width>341</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>First setting</string>
   </property>
   <widget class="QCheckBox" name="label_on">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>26</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Label on</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="syn_flag">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>22</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>22</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Syn_flag:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="syn_ref">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Peak</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Foot</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Syn_ref:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="data_show">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Show saved</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Show origin</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>960</width>
     <height>50</height>
    </rect>
   </property>
   <property name="title">
    <string>Data statistic</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Total files:</string>
    </property>
   </widget>
   <widget class="QLabel" name="total_file">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>15</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>15</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Total adjusted files:</string>
    </property>
   </widget>
   <widget class="QLabel" name="total_adjusted_file">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>15</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="control_group">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>460</y>
     <width>611</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Control</string>
   </property>
   <widget class="QLineEdit" name="save_link">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flush">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Flush</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restore">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Restore</string>
    </property>
   </widget>
   <widget class="QPushButton" name="packaging">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Package</string>
    </property>
   </widget>
   <widget class="QComboBox" name="package_option">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pakage_name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="valid">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>25</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Valid</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
